lib/request.js

```diff
@@ -29,9 +29,9 @@ const Request = {
     },
 
     protocol() {
-      return this.connection.encrypted
+      return (this.get('x-forwarded-proto') || this.connection.encrypted
```

Comment:
You don't need to wrap it in parens. Otherwise it looks good.

Reply:
Updated